distribution/pom.xml

```diff
@@ -164,6 +164,10 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
There's no licenses in documentation module. Only pom file has a header and all others - don't.